README.md

```diff
@@ -60,3 +60,7 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 #### how to run tests
 - ensure chromedriver is downloaded and added to system path (https://chromedriver.chromium.org/downloads) 
 - run `poetry run pytest` from the command line
+
+### Deployment
+The application is deployed to heroku when a new commit/merge is made to the `master` branch
```

Comment:
Your README still mentions `SECRET_KEY` which you don't use any more. Would be nice to briefly mention setting up the mongo connection variables in .env instead.

Reply:
updated